packages/react-image/Spec.md

```diff
@@ -0,0 +1,153 @@
+| Component design prop                                                   | N/A                     | `design`        | v7/v8 missing                 |
+| Image source URL                                                        | N/A                     | `src`           | v7/v8 missing                 |
+
+### v8 usage
+
+```jsx
+import { Image, ImageFit } from '@fluentui/react/lib/Image';
+
+<Image maximizeFrame imageFit={ImageFit.contain} src="..." alt="Example of the image component in v8" />;
+```
+
+### v0 usage
+
+```jsx
+<Image fluid src="..." alt="Example of the image component in v0" />
+```
+
+## Variants
+
+n/a
+
+## API proposal
+
+See [react-image/Image.types.ts] for the API.
+
+[react-image/image.types.ts]: https://github.com/microsoft/fluentui/blob/master/packages/react-image/src/components/Image/Image.types.ts
+
+| Name       | Type      | Default value | Comments                                     |
+| ---------- | --------- | ------------- | -------------------------------------------- |
+| `alt`      | `string`  | `undefined`   |                                              |
+| `bordered` | `boolean` | `false`       |                                              |
+| `fit`      | `string`  | "`none`"      | One of: `none`, `center`, `contain`, `cover` |
+| `fluid`    | `boolean` | `false`       |                                              |
+| `circular` | `boolean` | `false`       |                                              |
+| `rounded`  | `boolean` | `false`       | rounded corners                              |
+| `src`      | `string`  | `undefined`   |                                              |
+
+## Sample Code
+
+Sample code based on the proposed API:
+
+### Bordered
+
+```jsx
+<Image bordered src="..." />
+```
+
+### Fluid
+
+```jsx
+<Image fluid src="..." />
+```
+
+### Rounded
+
+```jsx
+<Image rounded src="..." />
+```
+
+### Circular
+
+```jsx
+<Image circular src="..." />
+```
+
+### Fit
+
+```jsx
+// None
+<Image fit="none" src="..." />
+
+// Center
+<Image fit="center" src="..." />
+
+// Contain
+<Image fit="contain" src="..." />
+
+// Cover
+<Image fit="cover" src="..." />
+```
+
+## Structure
+
+### Public
+
+```tsx
+<Image src=".." />
+```
+
+### Internal
+
+```tsx
+<slots.root {...slotProps.root} />
+```
+
+### DOM
+
+```tsx
+<img src="..." class="...">
+```
+
+## Migration
+
+<!-- _Describe what will need to be done to upgrade from the existing implementations:_
+
+- _Migration from v8_
+- _Migration from v0_ -->
+
+## Accessibility
+
+Images should always include the `alt` attribute along with a description of the content and function of the image to be accessible to screeen readers. Avoid using "image of", "picture of" etc. on the alt description. The images that are only decorative and do not have the structural relevance implied by the semantic element should use the role `presentation`.
```

Comment:
So `role="presentation"` might not be enough since some screen readers can still read it in virtual cursor mode. In this case `aria-hidden` should be set alongside the `role`

Reply:
I was not aware of this, thanks! Could you point out to any resource I could get more info about this? 
I will also ping Sarah for input regarding a11y for image and update this part to be more accurate.

Reply:
@smhigley can you provide more insights on this one pls 🙏 ?

Reply:
I researched about the differences between setting the role of presentation and adding the aria-hidden. The difference is that  the role presentation will only remove the sematic meaning of the element, whereas the aria-hidden will remove it from the tree. I did not find any source that confirms the above-mentioned issue with the role="presentation". 
[Here](https://w3c.github.io/aria/#presentation) is it specified that `role="presentation"` used in images is equivalent to `aria-hidden="true"`. According to the [MDN accessibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable#guideline_1.1_%E2%80%94_providing_text_alternatives_for_non-text_content), the recommended way is to add an empty `alt=""`.

I will continue with the implementation and I will test the accessibility behaviour of image to confirm this. I will modify the spec as needed.